### PR TITLE
Dam process options

### PR DESCRIPTION
**Pull request by DerekMayse**
****Things you should check/see****

- [x] Error message on main menu is now on a timer
- [x] Error message for create facility is now on a timer
- [x] fixed the grammar in farm report when you only have a single animal or plant 

For Example: "Natural Field has 1 plant" "Plowed Field has 2 plants"

- [x] added the process resources as an option on the main menu
- [x] the process resource menu displays 5 pieces of equipment 
- [x] invalid option will throw you an error message and direct you back to the main menu
- [x] each option displays a message for what option you picked then direct you back to the main menu

****Note****
* I believe i listed everything I've added or changed
* If I missed anything there should be comments in the code for what I did
* Anything else just let me know

### CODE REVIEW

**Review by dylbyl — APPROVED** [review]
All options seem to catch exceptions and errors as intended! Preliminary process menu also works as expected. Ready to merge after another approval

**Review by barrygriffith84 — APPROVED** [review]
- The timers on the error messages function correctly.
- The grammar has been corrected.
- The process resources option has been added to the main menu.
- The error messages for process resources are correct.
- The process resources options return the correct messages when selected.

Approved to merge.

**Pull request merged**
